### PR TITLE
fix(a11y): color contrast ratio fix

### PR DESCRIPTION
**Pull request by lyannel**
## Description of Changes

- Updated color for search placeholder text to fix a11y color contrast ratio issue

### Before Changes
Sass A11y Before Fix

### After Changes
SAS A11y After

### CODE REVIEW

File: source/assets/sass/components/_docsearch.scss [hunk 1]
```diff
@@ -4,3 +4,7 @@
   justify-content: center;
   height: 100%;
 }
+
+.DocSearch-Button-Placeholder {
+  color: #4A4F4F;
```
Comment (nex3) [inline source/assets/sass/components/_docsearch.scss:9]: Is there a named brand color we can use for this?

**Review by nex3 — COMMENTED** [review]
Thanks for taking the time to contribute!

File: source/assets/sass/components/_docsearch.scss (hunk 1 above)
Comment (lyannel) [reply source/assets/sass/components/_docsearch.scss]: Updated!

Comment (nex3) [reply source/assets/sass/components/_docsearch.scss]: I meant one of the brand colors from [the style guide](https://sass-lang.com/styleguide/color/).

Comment (lyannel) [reply source/assets/sass/components/_docsearch.scss]: Ah got it thank you-- updated!

**Comment by nex3** [comment]
Can you run prettier to fix the formatting?

**Comment by lyannel** [comment]
> Can you run prettier to fix the formatting?

@nex3 
sorry for late update!
ran the fix and updated 😄

**Review by nex3 — APPROVED** [review]

**Pull request merged**